client/main.lua

```diff
@@ -2,9 +2,30 @@ local config = require 'config.client'
 local JOBS = exports.qbx_core:GetJobs()
 local GANGS = exports.qbx_core:GetGangs()
 local isLoggedIn = LocalPlayer.state.isLoggedIn
+local dynamicMenuItems = {}
 
 lib.locale()
 
+-- Adds item to the boss/gang menu.
+---@param menuItem ContextMenuItem Requires args.type to be set to know which menu to place in.
+---@return number menuId ID of the menu item added
+local function addMenuItem(menuItem)
+    local menuId = #dynamicMenuItems + 1
+    if not menuId.args.type then return end
```

Comment:
menuId is a number, but is treated like a table here. Is this supposed to be menuItem? In which case move this line to be the first in the function.

Reply:
ah typo. my apologies